maintenance_plan/views/maintenance_plan_views.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="hr_equipment_request_action_from_plan" model="ir.actions.act_window">
+        <field name="name">Maintenance Requests</field>
+        <field name="res_model">maintenance.request</field>
+        <field name="binding_model_id" ref="model_maintenance_plan"/>
+        <field name="view_mode">kanban,tree,form,pivot,graph,calendar</field>
+        <field name="context">{
+            'default_maintenance_plan_id': active_id,
+        }</field>
+        <field name="domain">[('maintenance_plan_id', '=', active_id)]</field>
+    </record>
+
+    <record id="maintenance_plan_view_form" model="ir.ui.view">
+        <field name="name">maintenance.plan.form</field>
+        <field name="model">maintenance.plan</field>
+        <field name="arch" type="xml">
+            <form string="Maintenance Plan">
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
```

Comment:
This line needs to be updated, in Odoo 13 use:
``` xml
<widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
```
Outside the oe_button_box div

Otherwise this is the result:
![image](https://user-images.githubusercontent.com/44768500/72420040-47e7c500-377e-11ea-9cb8-7acc3cb3c5f5.png)